back/db/migrate/20240911143007_remove_polymorphism_from_official_feedback.rb

```diff
@@ -1,5 +1,12 @@
 class RemovePolymorphismFromOfficialFeedback < ActiveRecord::Migration[7.0]
   def change
+    execute <<~SQL.squish
```

Comment:
Without this part, deleting the official feedbacks can fail, when they are still being referenced by notifications (cleaning up those notifications would otherwise happen through a separate rake task, that is executed manually, so after this migration).